### ISSUE TITLE
Announcing "blank" for task bars without labels on Gantt Chart

### ISSUE THREAD

**Issue by taauntik** [bryntum/support#11657]
[Forum post](https://forum.bryntum.com/viewtopic.php?f=51&t=33591&p=171283#p171283)

Hi there,

We are encountering an announcement issue with the Bryntum Basic Gantt Chart. When focus shifts to a task bar, the screen reader announces "blank" instead of providing meaningful information for task bars with no label (For Example, parent rows).

Tested of below combinations -

Windows + chrome + NVDA
MAC + Chrome + VO
MAC + Safari + VO

![announcementBlank.png](https://forum.bryntum.com/download/file.php?id=35662)

Is there a recommended way to implement a custom announcement to improve accessibility for screen reader users, or is this something that should be addressed by Bryntum team?
![announcementBlank.png](https://forum.bryntum.com/download/file.php?id=35662)
